src/main/features/offer/guards/offerAcceptedGuard.ts

```diff
@@ -0,0 +1,26 @@
+import * as express from 'express'
+
+import { Paths } from 'dashboard/paths'
+import { Logger } from '@hmcts/nodejs-logging'
+import { Claim } from 'claims/models/claim'
+import { User } from 'idam/user'
+
+const logger = Logger.getLogger('offer/guards/offerAcceptedGuard')
+
+export class OfferAcceptedGuard {
+
+  static requestHandler (req: express.Request, res: express.Response, next: express.NextFunction): void {
```

Comment:
@ellie-harrison can you please use GuardFactory so that:
- we have consistent way of creating guards
- factory based way does not require any changes to a11y tests

Otherwise you would need to mocks your guard in a11y mocks.

See OnlyClaimantLinkedToClaimCanDoIt as an example.